solidity/app/deploy.js

```diff
@@ -0,0 +1,67 @@
+const Eth = require('ethjs')
+const TruffleContract = require('truffle-contract')
+const ABI = require('ethereumjs-abi')
+
+require('./cl_utils.js')
+require('./cl_wallet.js')
+const compile = require('./compile.js')
+
+function getBytecode (contract) {
+  return contract.evm.bytecode.object.toString()
+}
+
+function contractify (abi, address) {
+  let contract = TruffleContract({
+    abi: abi,
+    address: address
+  })
+  contract.setProvider(clUtils.provider)
+  contract.defaults({
+    from: clWallet.address,
+    gas: 6721975, // Truffle default
```

Comment:
Instead of comments how do you feel about extracting L#21 & L#22 into named variables? e.g. `const TRUFFLE_DEFAULT_GAS = 6721975 `, `const TRUFFLE_DEFAULT_GAS_PRICE = 100000000000 `?